addon/templates/components/new-bourbon-select-field.hbs

```diff
@@ -0,0 +1,12 @@
+<button class="BourbonSelectField-selected" aria-haspopup="listbox" aria-live="polite" aria-expanded={{ariaExpanded}} aria-activedescendant={{activeDescendant}} tabindex="0" {{action "mouseDown"}}>{{label}}</button>
```

Comment:
Do we need to set `{{action "mouseDown"}}` here? Just wondering if `mouseDown` could be defined as a component method and Ember could take care of it. The same way that Ember handles `focusOut` and `keyDown`.

Reply:
The reason why I needed this was because `mouseDown` on the whole element causes an issue where the users can't scroll the list which is an issue for our windows users.